.vscode/tasks.json

```diff
@@ -0,0 +1,107 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "process",
+			"label": "Build Terminal/Console",
+			"command": "powershell.exe",
+			"args": [
+			  "-Command",
+			  "Import-Module ${workspaceFolder}\\tools\\OpenConsole.psm1;",
+			  "Set-MsBuildDevEnvironment;",
+			  "$project = switch(\"${input:buildProjectChoice}\"){OpenConsole{\"Conhost\\Host_EXE\"} Terminal{\"Terminal\\CascadiaPackage\"}};",
+			  "$target = switch(\"${input:buildModeChoice}\"){Build{\"\"} Rebuild{\":Rebuild\"} Clean{\":Clean\"}};",
+			  "$target = $project + $target;",
+			  "msbuild",
+			  "${workspaceFolder}\\OpenConsole.sln",
+			  "/p:Configuration=${input:configChoice}",
+			  "/p:Platform=${input:platformChoice}",
+			  "/t:$target",
+			  "/verbosity:minimal"
+			],
+			"problemMatcher": ["$msCompile"],
+			"group": {
+			  "kind": "build",
+			  "isDefault": true
+			},
+			"runOptions": {
+				"reevaluateOnRerun": false,
+				"instanceLimit": 1,
+				"runOn": "default"
+			}
+		},
+		{
+			"type": "process",
+			"label": "Register Windows Terminal x64 Debug",
+			"command": "powershell.exe",
+			"args": [
+			  "-Command",
+			  "Import-Module ${workspaceFolder}\\tools\\OpenConsole.psm1;",
+			  "Set-MsBuildDevEnvironment;",
+			  "Set-Location -Path ${workspaceFolder}\\src\\cascadia\\CascadiaPackage\\AppPackages\\CascadiaPackage_0.0.1.0_x64_Debug_Test;",
+			  "if ((Get-AppxPackage -Name 'WindowsTerminalDev*') -ne $null) { Remove-AppxPackage 'WindowsTerminalDev_0.0.1.0_x64__8wekyb3d8bbwe'};",
```

Comment:
Dustin gave me a snippet to make this less big-hammer. Fully removing the package will stop/kill it, but it will also probably trash the settings making debugging that stuff hard.

Go find the snippet he gave you to just find the process and kill it instead here.

Reply:
```
get-process | where path -like "$(get-appxpackage microsoft.windowsterminalpreview |
select -expand installlocation)\*" | stop-process -force
```

Reply:
This no worky worky. Even with a 2 second sleep at the end, some of the files are still in-use.

Perhaps I leave this as is for now and someone else can refine further later?